src/main/java/DavinciBot.java

```diff
@@ -0,0 +1,276 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * DavinciBot is a simple bot that allows the user to manage a to-do list.
+ * The bot echoes commands entered by the user, adds tasks to the list,
+ * marks tasks as done, and unmarks tasks.
+ */
+public class DavinciBot {
+
+    private static final String LINE_SEPARATOR = "____________________________________________________________";
+    public static final int SPLIT_INTO_TWO_PARTS = 2;
+
+    /**
+     * Selects the icon corresponding to the type of task inputted by the user.
+     *
+     * @param taskArray Array of tasks.
+     */
+    private static void iconSelector(Task[] taskArray) {
```

Comment:
Function names should read as a verb. This name is not apt. Consider using something like `selectIcon` instead. 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions